### PR TITLE
Add src getter method

### PR DESCRIPTION
**Pull request by matthewsuan**
Allow developers to retrieve the src.

```
$mediaEmbed = new MediaEmbed;
$object = $mediaEmbed->parseUrl($rl);
$object->getEmbedSrc(); // https://www.youtube.com/embed/qzBGa0OAbfE
```

### CODE REVIEW

**Comment by dereuromark** [comment]
Looks good. I might have to release a 0.5 here then.

**Comment by dereuromark** [comment]
A small test case would be good to show that this works.

File: .phpunit.result.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...t::testParseUrl with data set #36";d:0.001;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #37";d:0.002;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #38";d:1.48;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #39";d:1.181;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #40";d:2.124;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #41";d:0.002;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #42";d:0.001;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #43";d:0.001;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #44";d:0.001;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #45";d:0.001;s:62:"MediaEmbed\Test\MediaEmbedTest::testParseUrl with data set #46";d:0.001;s:43:"MediaEmbed\Test\MediaEmbedTest::testParseId";d:0.001;s:43:"MediaEmbed\Test\MediaEmbedTest::testYoutube";d:0.211;s:56:"MediaEmbed\Test\MediaEmbedTest::testYoutubeWithoutIframe";d:0.001;s:47:"MediaEmbed\Test\MediaEmbedTest::testDailymotion";d:0.001;s:46:"MediaEmbed\Test\MediaEmbedTest::testMatterport";d:0.002;s:44:"MediaEmbed\Test\MediaEmbedTest::testGetHosts";d:0.001;}}}
```
Comment (dereuromark) [inline .phpunit.result.cache]: this file should be in gitignore

**Comment by matthewsuan** [comment]
Thanks!

**Pull request merged**